data-structures-stacks/get-2nd-from-top.js

```diff
@@ -0,0 +1,16 @@
+/* exported get2ndFromTop */
+
+function get2ndFromTop(stack) {
+  if (stack.peek() === undefined) {
+    return undefined;
+  }
+  const poppedVal = stack.pop();
+  if (stack.peek() === undefined) {
+    stack.push(poppedVal);
+  } else if (stack.peek() !== undefined) {
```

Comment:
Just an `else` here would be better, there is no need to check the exact opposite of what you already checked, that's what the `else` is for.

Reply:
Thank you Scott.